app/controllers/api_news_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ApiNewsController < ApplicationController
+  
+  def index
```

Comment:
I guess I still have the question of whether this could go in the articles controller somewhere, but I think for now it makes sense to leave here and perhaps refactor (or get Thomas's view when he reviews PR).